libafl_frida/src/coverage_rt.rs

```diff
@@ -15,15 +22,19 @@ use crate::helper::FridaRuntime;
 /// (Default) map size for frida coverage reporting
 pub const MAP_SIZE: usize = 64 * 1024;
 
-/// Frida binary-only coverage
 #[derive(Debug)]
-pub struct CoverageRuntime {
+struct CoverageRuntimeInner {
     map: [u8; MAP_SIZE],
     previous_pc: u64,
     current_log_impl: u64,
     blob_maybe_log: Option<Box<[u8]>>,
+    _pinned: PhantomPinned,
 }
 
+/// Frida binary-only coverage
+#[derive(Debug)]
+pub struct CoverageRuntime(Pin<Arc<RefCell<CoverageRuntimeInner>>>);
```

Comment:
Does this need to be threadsafe? Else we should use `Rc` instead of `Arc` for increased performance here

Reply:
I guess not? Each thread has its own instance

Reply:
Then let's do `Rc` :) it's the same thing, just not atomic